src/neuroconv/basedatainterface.py

```diff
@@ -79,7 +79,8 @@ def run_conversion(
             warnings.warn(
                 "Using DataInterface.run_conversion() without specifying nwbfile_path is deprecated. To create an "
                 "NWBFile object in memory, use DataInterface.create_nwbfile(). To append to an existing NWBFile object,"
-                " use DataInterface.add_to_nwbfile()."
+                " use DataInterface.add_to_nwbfile().",
+                stacklevel=2,
```

Comment:
Found this useful in a separate conversion project; kept getting the warning but was unsure where it was coming from (had several data interfaces in the conversion)